examples/advanced/llm_hf/README.md

```diff
@@ -0,0 +1,131 @@
+# LLM Tuning via HuggingFace SFT Trainer
+This example shows how to use [NVIDIA FLARE](https://nvidia.github.io/NVFlare) for Large Language Models (LLMs) tuning tasks. It illustrates how to adapt a local training script with [HuggingFace](https://huggingface.co/) trainer to NVFlare.
+
+## Introduction 
+This example illustrates both supervised fine-tuning (SFT) and parameter-efficient fine-tuning (PEFT) using the [SFT Trainer](https://huggingface.co/docs/trl/sft_trainer) from [HuggingFace](https://huggingface.co/).
+We used the [Llama-2-7b model](https://huggingface.co/meta-llama/Llama-2-7b) to showcase the functionality of federated SFT and PEFT, allowing HuggingFace models to be trained and adapted with NVFlare. 
+Mainly on two fronts:
+- Adapt local HF training scripts to federated application
+- Handling large model weights (~26 GB for Llama-2-7b model), this is supported by NVFlare infrastructure, and does not need any code change.
+
+We perform the following exeperiments on two 80GB A100 GPUs, PEFT only needs 1 GPU, while SFT needs both GPUs. Less computation resources will be needed if smaller models are used, simply replace Llama-2-7b-hf with other options from HuggingFace.
```

Comment:
Should be just "Llama-2-7b" instead of "Llama-2-7b-hf"?

Reply:
otherwise suggest rename all "Llama-2-7b" to "Llama-2-7b-hf"

Reply:
-hf means "huggingface format", models without -hf are intended for native llama-2 code

Reply:
change all to -hf